src/playground/service.clj

```diff
@@ -74,9 +74,9 @@
     ["/about" :get (conj common-interceptors `about-page)]
     ["/api" :get (into component-interceptors [http/json-body (param-spec-interceptor ::api :query-params) `api])]
     ["/invoices/insert" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.insert/api :query-params) `invoices.insert/perform])]
-    ["/invoices/:id" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.retrieve/api :path-params) `invoices.retrieve/perform])]
+    #_["/invoices/:id" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.retrieve/api :path-params) `invoices.retrieve/perform])]
```

Comment:
ignoring this form allows : `http://localhost:8080/invoices/insert?amount=22` to be inserted to the table.

Reply:
But such routing isn't twice here https://github.com/reducecombine/playground/blob/master/src/playground/service.clj

Routes can have exactly one entry per path (`/invoices/:id`) and method (`POST`), else how would the router know what option to pick?

Reply:
Ah no sorry it's not twice, misread the diff. Ignore my comment